challenges.html

```diff
@@ -93,9 +104,12 @@ <h2 class="booking-modal__title">Enter Your Details (Step2/Modal2)</h2>
               
                             <!-- Step 3: Confirmation -->
                             <div class="booking-modal__step" id="step3">
-                              <h2 class="booking-modal__title">Thank You!</h2>
-                              add a paragraph for the confirmation and a button to close the modal button
-                            </div>
+                              <div class="modal" id="modal3">
+                                <div class="modal__content">
+                                    <h1>Thank you!</h1>
+                                    <a href="index.html">Back to challenges</a>
```

Comment:
It would be amzing if you redirect the the user to challenges.html as it is a requirement  for the assignment <3 'När användaren går vidare ska ett tackmeddelande visas, samt möjlighet att stänga ned rutan och återgå till listan över utmaningar'